README.md

```diff
@@ -17,43 +17,40 @@ in the scripts folder and understand as much as possible. We'll touch some
 things briefly here that are better explained there. The steps here will expect
 you have some knowledge of what's going on inside the script.
 
-  1. Checkout LLVM to a to your home directory from
-     [OUR FORK](https://github.com/cmput415/llvm-project), checkout the 10.0.0
-     release, and change the directory in your script.
+  1. Checkout LLVM to your machine
      1. `cd $HOME`
-     1. `git clone git@github.com:cmput415/llvm-project.git`
-     1. `cd llvm`
-     1. `git checkout llvmorg-10.0.0`
-  1. Add these configuration lines to your `~/.bashrc` on linux or
-     `~/.bash_profile` on MacOS to setup the next steps. You should restart your
+     1. `git clone https://github.com/llvm/llvm-project.git`
+     1. `cd llvm-project`
+     1. `git checkout llvmorg-16.0.4`
+  1. Build MLIR
+     1. `mkdir build`
+     1. `cd build`
+     1. ```cmake -G Ninja ../llvm \
+             -DLLVM_ENABLE_PROJECTS=mlir \
+             -DLLVM_BUILD_EXAMPLES=ON \
+             -DLLVM_TARGETS_TO_BUILD="Native;NVPTX;AMDGPU" \
+             -DCMAKE_BUILD_TYPE=Release \
+             -DLLVM_ENABLE_ASSERTIONS=ON
+        ```
+     1. `cmake --build . --target check-mlir`
+  1. Add these configuration lines to your ``~/.bashrc`` on linux or
+     ``~/.zprofile`` on MacOS to setup the next steps. You should restart your
      terminal after editing these files.
       ```bash
-      export LLVM_INS="$HOME/llvm-ins/" # Change me if you really want.
-      export LLVM_DIR="$LLVM_INS/lib/cmake/llvm/" # Don't change me.
-      export PATH="$LLVM_INS/bin:$PATH" # Don't change me
+      export MLIR_INS="$HOME/llvm-project/build/"
+      export MLIR_DIR="$MLIR_INS/lib/cmake/mlir/" # Don't change me.
+      export PATH="$MLIR_INS/bin:$PATH" # Don't change me
       ```
-  1. If you want a debug build you may change this by looking at Step 2 in the
-     `scripts/configureLLVM.sh` script.
-  1. Run `configureLLVM.sh`.
```

Comment:
Is this script just fully cut? It's still in the repo though.

Reply:
I left it in because we may want to adapt it to MLIR. It's good because it explains all the different options that the simple one in README.md uses.